dz-12/src/test/java/cc/robotdreams/tests/ManAndWomanTests.java

```diff
@@ -0,0 +1,137 @@
+package cc.robotdreams.tests;
+import cc.robotdreams.Man;
+import cc.robotdreams.Woman;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+@Test
+      public class ManAndWomanTests {
+
+        private enum ExpectedStatus
+        {
+         PASSED,
+         FAILED
+        }
+
+        @DataProvider
+        public Object[][] DataMan() {
+            return new Object[][]{
+                    {ExpectedStatus.PASSED,"Joe","Doe", 70},
```

Comment:
Better to use structure like `{ExpectedStatus.PASSED, new Man("Joe","Doe", 70)}`